docs/porting-guide.rst

```diff
@@ -52,7 +59,6 @@ Import statements will need to change:
     # note that the original "Event" has been decommissioned
     from cbc_sdk.endpoint_standard import Device, EnrichedEvent, Policy
```

Comment:
While we're updating, could we change this to use the platform Device and Policy classes, and also a note to consider using Observations instead of Enriched Events.   Blog about the benefits of Observations is here to add a bit of context or reference something: https://developer.carbonblack.com/2023/07/how-to-take-advantage-of-the-new-observations-api/.  One of the other benefits is that they apply to all products, not only endpoint standard.

from cbc_sdk.endpoint_standard import EnrichedEvent
from cbc_sdk.platform import Device, Policy

Reply:
I'm going to have to think about this one, especially since I don't know Observations.

Reply:
Scratch that last. Looks like all I need to do is change up some imports.